pkg/config/config.go

```diff
@@ -105,6 +105,9 @@ type Server struct {
 	Auth Auth `mapstructure:"auth"`
 
 	MetricExportRules MetricExportRules `yaml:"metric-export-rules" def:"" desc:"metric export rules" mapstructure:"metric-export-rules"`
+
+	TLSCertificateFile    string `def:"" desc:"location of TLS Certificate file (.crt)" mapstructure:"tls-certificate-file"`
+	TLSCertificateKeyFile string `def:"" desc:"location of TLS Private key file (.key)" mapstructure:"tls-certificate-key-file"`
```

Comment:
I know this is pretty minor but I'd just name this one `TLSKeyFile`:
```suggestion
	TLSKeyFile string `def:"" desc:"location of TLS Private key file (.key)" mapstructure:"tls-key-file"`
```

It's shorter plus I think this is more common, e.g:
* in [go's ListenAndServeTLS](https://cs.opensource.google/go/go/+/refs/tags/go1.17.1:src/net/http/server.go;l=3192) variables are named certFile and keyFile
* in [redis](https://github.com/redis/redis/blob/unstable/redis.conf#L168-L169) it's `tls-cert-file` and `tls-key-file`

Reply:
some bad muscle memory from configuring apache's `SSLCertificateKeyFile` :)